src/main/java/emu/grasscutter/game/combine/CombineManger.java

```diff
@@ -29,16 +30,11 @@
 
 import com.google.gson.reflect.TypeToken;
 
-public class CombineManger {
-    private final GameServer gameServer;
+public class CombineManger extends BaseGameSystem {
```

Comment:
```suggestion
public class CombineSystem extends BaseGameSystem {
```
(and all usages of it)